test/e2e/framework/pod/resize.go

```diff
@@ -243,63 +193,72 @@ func VerifyPodStatusResources(gotPod *v1.Pod, wantCtrs []ResizableContainerInfo)
 	return utilerrors.NewAggregate(errs)
 }
 
+func verifyContainerCgroupValues(f *framework.Framework, pod *v1.Pod, tc *v1.Container, podOnCgroupv2 bool) error {
```

Comment:
should we move this function and below ones to the new `cgroups.go` file? mainly `verifyContainerCgroupValues`, `buildPodResourceInfo` and `VerifyPodContainersCgroupValues` funcs.